### DISCUSSION TITLE
Using exclude directories does not seem to work the same with the new Uno.Sdk

### DISCUSSION THREAD

**Discussion by cconner100** [unoplatform/uno#15298] [General]
I am trying to convert my existing project to use the new Uno.SDK.

In my previous project i was able to put in an exclude of a directory at the bottom of the csproj file.  It worked fine but not it does not seem to work

attached is a small repro project.  It uses the new wizzard and only implements Windows and Android.  Under presentations you will see 2 directories, Windows and Android.  Windows has windows specific code in the xaml.

This use to work but maybe i am now missing something new?

[SdkRepro.zip](https://github.com/unoplatform/uno/files/14118501/SdkRepro.zip)

**Comment by jeromelaban** [comment]
Thanks for the report. Let's keep track in https://github.com/unoplatform/uno/issues/15321.
